exotations/solvers/exotica_levenberg_marquardt_solver/package.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>exotica_lm_solver</name>
+  <version>1.0.1</version>
+  <description>A Levenberg-Marquardt solver for EXOTica</description>
+
+  <maintainer email="Christian.Rauch@ed.ac.uk">Christian Rauch</maintainer>
+
+  <license>MIT</license>
```

Comment:
FYI: The rest of the repository is BSD licensed - both are compatible though so no action required.

Reply:
I like the MIT because it's shorter :-)